vault/logical_system_quotas.go

```diff
@@ -354,9 +360,17 @@ func (b *SystemBackend) handleRateLimitQuotasUpdate() framework.OperationFunc {
 			return nil, err
 		}
 
+		inheritable := false
+		inheritableI, ok := d.GetOk("inheritable")
+		if ok {
+			inheritable = inheritableI.(bool)
```

Comment:
(nit) For consistency:

```suggestion
		inheritableRaw, ok := d.GetOk("inheritable")
		if ok {
			inheritable = inheritableRaw.(bool)
```